docs/assert/true.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: true
+description: A boolean check, equivalent to Chai's assert.isTrue() and JUnit's assertTrue(). Passes if the first argument is true.
+categories:
+  - assert
+redirect_from:
+  - "/true/"
```

Comment:
Many files contain this for compat with URLs from an older iteration of the API doc site (which was based on WordPress and jquer-wp-content). Not needed for new pages. I'll omit this while landing.